backend/src/main/scala/org/soaringmeteo/gfs/out/Winds.scala

```diff
@@ -0,0 +1,63 @@
+package org.soaringmeteo.gfs.out
+
+import org.soaringmeteo.Wind
+import squants.space.{Length, Meters}
+
+import scala.annotation.tailrec
+import scala.collection.immutable.ArraySeq
+
+/** Wind values at several elevation levels */
+case class Winds(
+  `300m AGL`: Wind,
+  boundaryLayerTop: Wind
+)
```

Comment:
Not sure what the most relevant elevation levels could be? We have surface wind, average boundary layer wind, boundary layer top wind, and 300 m AGL.

Reply:
Very nice. 
Would it be possible to have 300m AGL 1500m MSL 2500m MSL for example? The boundary top depends on the day, requiring a bit of calculation :-)

If someone opens the page has has not yet made a selection, how about:
a) showing only the wind at 300m AGL initially (less cluttered)
b) automatically open the setting box on the right, so the user can immediately see the options available?

We have not yet linked this v2 on the site? A pity, at least put it in the News section?

Reply:
> Would it be possible to have 300m AGL 1500m MSL 2500m MSL for example? The boundary top depends on the day, requiring a bit of calculation :-)

Yes it is possible, but personally, I don’t think it is very useful to see the wind at fixed altitudes AMSL. If I see the wind at 2500 m AMSL but the the boundary layer top is at 2000 m, then I’m seeing wind at an altitude I can’t reach with my paraglider, so, what’s the point? If I see the wind at 1500 m AMSL, then it is not relevant for the alps because the ground is often higher than 1500 m.

That was my reasoning for showing the wind AGL.

I think the average wind in the boundary layer is useful because that’s the area we fly in. The top of the boundary layer is useful too because that’s often the strongest wind we can get by paragliding. Finally, I thought 300 m AGL would be useful because that’s typically the altitude we fly (±200 m).

Reply:
> If someone opens the page has not yet made a selection, how about:
> a) showing only the wind at 300m AGL initially (less cluttered)
> b) automatically open the setting box on the right, so the user can immediately see the options available?

I definitely agree that the UX can be improved. Unfortunately, I don’t have the bandwidth to do many changes in this area.

> We have not yet linked this v2 on the site? A pity, at least put it in the News section?

No, maybe it’s time to merge https://github.com/Boran/soaringmeteo-website/pull/32?

Reply:
merged #32.

"I don’t think it is very useful to see the wind at fixed altitudes AMSL"
When I plan for the weekend I look at 1500, 2500, 3500 to choose where I would be able to take off (LP, préalpes, haut alpes), and top get a feel for the wind flow. I use meteoblue for this.
Analysis of the boundary layer to seer how high one can fly and it what direction is second step.

300AGL is a good generic measurement through.